client/verta/tests/test_dataset_versioning/test_dataset_version.py

```diff
@@ -18,6 +19,10 @@ def test_description(self, client, dataset, strs):
         assert version.get_description() == second_desc
 
         assert dataset.get_version(id=version.id).get_description() == second_desc
+        client.set_workspace("42379216512358509179263")
+        dataset = client.create_dataset("dataset_name3")
+        version = dataset.create_version(Path("conftest.py"), desc=first_desc)
```

Comment:
Should this be removed? I'm not sure if it is a part of the test, or if it was for debugging.